File: benchmarking/GenSSI/NFkB.m
function model = SMTH()
syms x15 x13 x7 x9 x2 x8 x14 x5 x4 x12 x6 x1 x10 x11 x3
syms e2a i1 i1a c5 k_deg k2 t2 c3a k3 k1 c4a k_prod t1
syms x150 x130 x70 x90 x20 x80 x140 x50 x40 x120 x60 x10 x100 x110 x30
syms u
model.sym.p = [e2a; i1; i1a; c5; k_deg; k2; t2; c3a; k3; k1; c4a; k_prod; t1; x150; x130; x70; x90; x20; x80; x140; x50; x40; x120; x60; x10; x100; x110; x30];
model.sym.x = [x15; x13; x7; x9; x2; x8; x14; x5; x4; x12; x6; x1; x10; x11; x3];
model.sym.g = [0; 0; 0; 0; 0; 0; 0; 0; 0; 0; 0; -k1 * x1; 0; 0; 0];
model.sym.x0 = [x150; x130; x70; x90; x20; x80; x140; x50; x40; x120; x60; x10; x100; x110; x30];
model.sym.xdot = [-1/2500*x15 + 1/2000000*x7
e2a*x14 + 1/2*x10*x6 - x2*x13 - 1/50000*x13
5*i1*x6 - 1/2*x11*x7
-1/2500*x9 - 1/2000000*x7
-1/5*x10*x2 - x2*x13 - x2*k_deg - x2*k2*u*x8 - x2*k3 + x1*u*k1 + x4*t1 + x5*t2
1/2*x9 - c5*x8
-5*e2a*x14 + 1/2*x11*x7
x2*x13 - x5*t2
1/5*x10*x2 - x4*t1
1/2000000*x7 - c3a*x12
-1/2*x10*x6 + 1/50000*x13 - i1*x6 - x5*t2
-x1*k_deg - x1*u*k1 + k_prod
-1/5*x10*x2 - x10*i1a - 1/2*x10*x6 - 1/10000*x10 + 1/2000*x11 + x12*c4a
5*x10*i1a - 1/2*x11*x7 - 1/400*x11
x2*k2*u*x8 + x2*k3 - x3*k_deg];
model.sym.y = [x2 + x3 + x1
x12
x10 + x13
x7
x9
x2];
end

File: benchmarking/IdentifiableFunctions/results/NFkB/NFkB.jl
# NFkB
#! format: off
using StructuralIdentifiability

system = @ODEmodel(
	x8'(t) = 1//2*x9(t) - c5*x8(t),
	x3'(t) = x2(t)*k2*u(t)*x8(t) + x2(t)*k3 - x3(t)*k_deg,
	x10'(t) = -1//5*x10(t)*x2(t) - x10(t)*i1a - 1//2*x10(t)*x6(t) - 1//10000*x10(t) + 1//2000*x11(t) + x12(t)*c4a,
	x1'(t) = -x1(t)*k_deg - x1(t)*u(t)*k1 + k_prod,
	x12'(t) = 1//2000000*x7(t) - c3a*x12(t),
	x9'(t) = -1//2500*x9(t) - 1//2000000*x7(t),
	x6'(t) = -1//2*x10(t)*x6(t) + 1//50000*x13(t) - i1*x6(t) + x5(t)*t2,
	x4'(t) = 1//5*x10(t)*x2(t) - x4(t)*t1,
	x7'(t) = 5*i1*x6(t) - 1//2*x11(t)*x7(t),
	x11'(t) = 5*x10(t)*i1a - 1//2*x11(t)*x7(t) - 1//400*x11(t),
	x5'(t) = x2(t)*x13(t) - x5(t)*t2,
	x14'(t) = -5*e2a*x14(t) + 1//2*x11(t)*x7(t),
	x15'(t) = -1//2500*x15(t) + 1//2000000*x7(t),
	x13'(t) = e2a*x14(t) + 1//2*x10(t)*x6(t) - x2(t)*x13(t) - 1//50000*x13(t),
	x2'(t) = -1//5*x10(t)*x2(t) - x2(t)*x13(t) - x2(t)*k_deg - x2(t)*k2*u(t)*x8(t) - x2(t)*k3 + x1(t)*u(t)*k1 + x4(t)*t1 + x5(t)*t2,
	y3(t) = x9(t),
	y6(t) = x12(t),
	y2(t) = x10(t) + x13(t),
	y4(t) = x2(t) + x3(t) + x1(t),
	y5(t) = x2(t),
	y1(t) = x7(t)
)
